src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
+ *  readOnlyControl
+ *  itemsetData
```

Comment:
I think this one is specific to the itemsets plugin, so probably should not be listed here.